pages/performance/what.md

```diff
@@ -5,4 +5,6 @@ category: Getting started
 layout: styleguide
 ---
 
-TKTKTK
+Web Performance is the speed at which a website loads and appears in your browser when you type in the URL. A site’s performance can be good or bad. Good performance means your users can interact with your site easily, while bad performance can cause your site to not open or your users to leave without taking any action.
```

Comment:
Maybe this?

Web performance generally refers to how quickly a URL loads and appears in a web browser. Sites are considered to have  good performance if they load quickly and allow users to easily interact with them. Sites with poor performance typically have slow, or halting, page loads and interaction delays, which may prevent users from accessing your content or using your web applications.